package.json

```diff
@@ -17,7 +17,9 @@
     "postcss-media-query-parser": "^0.2.3",
     "postcss-resolve-nested-selector": "^0.1.1",
     "postcss-selector-parser": "^6.1.0",
-    "postcss-value-parser": "^4.2.0"
+    "postcss-value-parser": "^4.2.0",
+    "css-tree": "2.3.1",
+    "is-plain-object": "5.0.0"
```

Comment:
Can you please move these new dependencies to be before `known-css-properties` and run `npm install` to update `package-lock.json`. NPM will anyway re-order the dependencies when something is removed or added to dependencies.